notebooks/non-sequential.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>I appreciate the motivation coming from the data/data analysis questions. One idea/question to think about here/with the other notebooks as well - should we agree on one data set and use that consistently across the tutorial?  Currently I think we all use different data sets. Might be more continuity if we tell a common data analysis question and continually improve on it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>